pkg/tagger/subscriber/subscriber.go

```diff
@@ -54,6 +56,8 @@ func (s *Subscriber) Unsubscribe(ch chan []types.EntityEvent) {
 
 	delete(s.subscribers, ch)
 	close(ch)
+
+	telemetry.Subscribers.Dec()
```

Comment:
maybe it can be  line 56
```suggestion
	defer telemetry.Subscribers.Dec()
```